products/memgraph/v0.51.0/how_to_guides/use-query-modules-provided-by-memgraph.md

```diff
@@ -51,17 +51,182 @@ loaded modules:
+
+```bash
++-------------------------------------------+-------------------------+
+| name                                      | value                   |
++-------------------------------------------+-------------------------+
+| "Number of nodes"                         | "7"                     |
+| "Number of edges"                         | "6"                     |
+| "Number of bridges"                       | "6"                     |
+| "Number of articulation points"           | "2"                     |
+| "Average degree"                          | "0.8571428571428571"    |
+| "Sorted nodes degree"                     | "[(16, 4),(19, 3), ..." |
+| "Self loops"                              | "0"                     |
+| "Is bipartite"                            | "True"                  |
+| "Is planar"                               | "True"                  |
+| "Is biconnected"                          | "False"                 |
+| "Is weakly connected"                     | "True"                  |
+| "Number of weakly connected components"   | "1"                     |
+| "Is strongly connected"                   | "False"                 |
+| "Number of strongly connected components" | "7"                     |
+| "Is DAG"                                  | "True"                  |
+| "Is eulerian"                             | "False"                 |
+| "Is forest"                               | "True"                  |
+| "Is tree"                                 | "True"                  |
++-------------------------------------------+-------------------------+
+```
+
+#### Page Rank
+
+To run [Page Rank](https://en.wikipedia.org/wiki/PageRank) on top of data
+stored inside Memgraph, the following graph might be used:
+
+![](../data/pagerank_graph.png)
+
+To load the graph into Memgraph, the following query should be used:
+
+```opencypher
+CREATE (na {name: "Page A"})
+CREATE (nb {name: "Page B"})
+CREATE (nc {name: "Page C"})
+CREATE (nd {name: "Page D"})
+CREATE (na)-[:e]->(nb)
+CREATE (na)-[:e]->(nc)
+CREATE (nc)-[:e]->(na)
+CREATE (nb)-[:e]->(nc)
+CREATE (nd)-[:e]->(nc);
+```
+
+By executing `pagerank.pagerank()`, Memgraph will return the rank for each
+node as follows:
+
+```opencypher
+CALL pagerank.pagerank() YIELD *;
++--------------------+----------+
+| node               | rank     |
++--------------------+----------+
+| ({name: "Page C"}) | 0.39415  |
+| ({name: "Page D"}) | 0.0375   |
+| ({name: "Page A"}) | 0.372526 |
+| ({name: "Page B"}) | 0.195824 |
++--------------------+----------+
+```
+
+NOTE: A documented list of Page Rank parameters is located inside `pagerank.py`
+file installed with your Memgraph package under
+`/usr/lib/memgraph/query_modules`.
+
+#### Weakly Connected Components
+
+`wcc.py` query module can run
+[WCC](https://mathworld.wolfram.com/WeaklyConnectedComponent.html) analysis on
+top of a sub-graph. To get the number of weakly connected components and nodes
```

Comment:
I would remove the `on top of` wording.
Just say: ...analysis on a sub-graph of the whole graph.